pmmlserver/dummy_pyproject.toml

```diff
@@ -0,0 +1,11 @@
+[tool.poetry]
+name = "workaround-for-editable-install"
+version = "0.0.1"
+description = ""
+authors = ["none"]
+
+[tool.poetry.dependencies]
+python = ">=3.8,<3.12"
```

Comment:
Let's use the same upper boundary in all `pyproject.toml` files, unless there is a clear difference between them.

Check also [this comment](https://github.com/canonical/kserve-rocks/pull/20/files#r1498862246)

Reply:
```suggestion
# This is inherited from https://github.com/kserve/kserve/blob/v0.11.2/python/pmmlserver/pyproject.toml#L13
python = ">=3.8,<3.12"
```